### PR TITLE
Ansible OMERO Galaxy roles: switch to new namespace

### PR DESCRIPTION
**Pull request by sbesson**
See also https://github.com/ome/ansible-example-omero-onenode/pull/1 and https://github.com/IDR/deployment/pull/194

Switch all roles associated with the installation of OMERO.server and OMERO.web to use the roles published under the Galaxy `ome` namespace (following the deprecation of the `openmicroscopy` namespace).

Most roles are bumped to the next patch release which should be harmless. Only exceptions are `ome.ice 2.1.1 -> 3.0.0` (drop support for Ice 3.5) and `ome.omero_python_deps 1.1.0 -> 2.0.1` (remove Scipy)

To be run in check-mode against:

- [x] ome-dundeeomero

   ```
   (ansible) sbesson@ls30630:prod (master) $ ansible-playbook -KC playbooks/ome-dundeeomero.yml 
   ...
   PLAY RECAP *********************************************************************************************************************************************
   ome-dundeeomero.openmicroscopy.org : ok=75   changed=3    unreachable=0    failed=0   
   ```

   Changes related to pip in check-mode and the figure script

- [x] ome-demoservers

   ```
   (ansible) sbesson@ls30630:prod (master) $ ansible-playbook -KC playbooks/ome-demoserver.yml
   ...
   PLAY RECAP *********************************************************************
   ns-web-pub.openmicroscopy.org : ok=4    changed=0    unreachable=0    failed=0   
   ns-web.openmicroscopy.org  : ok=4    changed=0    unreachable=0    failed=0   
   ome-demoserver.openmicroscopy.org : ok=137  changed=11   unreachable=0    failed=0   
   pub-omero.openmicroscopy.org : ok=140  changed=10   unreachable=0    failed=0 
   ```
  
  Changes are related to Nginx, pip/virtualenv and the figure scripts

- [X] ns-webclients

   ```
   (ansible) sbesson@ls30630:prod (master) $ ansible-playbook -KC playbooks/nightshade-webclients.yml 
   ...
   PLAY RECAP *********************************************************************************************************************************************
   ns-web-pub.openmicroscopy.org : ok=74   changed=7    unreachable=0    failed=0   
   ns-web.openmicroscopy.org  : ok=74   changed=7    unreachable=0    failed=0   
   ome-demoserver.openmicroscopy.org : ok=4    changed=0    unreachable=0    failed=0   
   pub-omero.openmicroscopy.org : ok=4    changed=0    unreachable=0    failed=0 
   ```
  
  Changes are related to Nginx, pip/virtualenv in check-mode

- [x] outreach
 
  ```
   (ansible) sbesson@ls30630:prod (master) $ ansible-playbook -KC playbooks/omero/training-server/playbook.yml 
   ...
   PLAY RECAP *********************************************************************************************************************************************
   outreach.openmicroscopy.org : ok=131  changed=14   unreachable=0    failed=0   
  ```

   Changes in pip/virtualenv (check-mode), and scripts download

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [TRAINING-merge#331](https://ci.openmicroscopy.org/job/TRAINING-merge/331/). See the [console output](https://ci.openmicroscopy.org/job/TRAINING-merge/331/consoleText) for more details.
Possible conflicts:
  - PR #186 manics 'ome-training-2: Replacement for current outreach'
    - requirements.yml
    - omero/training-server/playbook.yml

--conflicts

**Comment by joshmoore** [comment]
Code certainly looks good. Can you summarize what you think is covered by travis and what isn't?

**Comment by sbesson** [comment]
I also asked myself the question. My understanding is that in Travis CI:

- the `omero-training` playbook is quite thoroughly covered by a complete `molecule` lifecycle
- all the other OMERO playbooks are passing a minimal syntax-check

**Comment by sbesson** [comment]
Deployed on `test70`

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [TRAINING-merge#332](https://ci.openmicroscopy.org/job/TRAINING-merge/332/). See the [console output](https://ci.openmicroscopy.org/job/TRAINING-merge/332/consoleText) for more details.
Possible conflicts:
  - PR #186 manics 'ome-training-2: Replacement for current outreach'
    - requirements.yml
    - omero/training-server/playbook.yml

--conflicts

**Comment by manics** [comment]
Have you checked site.yml in check-mode too?

**Comment by sbesson** [comment]
Ah thanks for poiting me at `site.yml` as it forced me to bump the Prometheus and Nginx proxy roles to avoid dependency conflicts. In check-mode, I have now:

```
PLAY RECAP *********************************************************************************************************
ns-web-pub.openmicroscopy.org : ok=101  changed=7    unreachable=0    failed=0   
ns-web.openmicroscopy.org  : ok=101  changed=7    unreachable=0    failed=0   
ome-demoserver.openmicroscopy.org : ok=181  changed=14   unreachable=0    failed=0   
ome-dundeeomero.openmicroscopy.org : ok=110  changed=6    unreachable=0    failed=0   
ome-training-jh.openmicroscopy.org : ok=57   changed=6    unreachable=0    failed=0   
outreach.openmicroscopy.org : ok=172  changed=15   unreachable=0    failed=0   
pub-omero.openmicroscopy.org : ok=187  changed=14   unreachable=0    failed=0 
```

Most changes are pertaining to pip and custom scripts installations
